### DISCUSSION TITLE
Javascript file truncates on save

### DISCUSSION THREAD

**Discussion by khoa-l** [LazyVim/LazyVim#1361] [Q&A] [answered]
I don't know if this is a LazyVim specific question, but was hoping to get pointed in the right direction! Working with a Javascript file and occasionally saving the file truncates it and deletes content below a certain point. I have it setup with pretterd and tsserver in Mason and tsserver in the nvim-lspconfig servers list, and that's about it.

https://github.com/LazyVim/LazyVim/assets/74576123/98b8c5d8-9050-401e-af5d-5761655688b7

**Comment by dpetka2001** [comment, marked as answer]
There have been other complaints about `prettierd` truncating files as a `null-ls` source. The solution was to use `prettier` instead. See #712 for reference.

**Reply by khoa-l** [reply]
Thank you!
